ckanext/stadtzhtheme/dcat/profiles.py

```diff
@@ -0,0 +1,272 @@
+    'adms': ADMS,
+    'vcard': VCARD,
+    'foaf': FOAF,
+    'schema': SCHEMA,
+    'time': TIME,
+    'skos': SKOS,
+    'locn': LOCN,
+    'gsp': GSP,
+    'owl': OWL,
+    'xml': XML,
+}
+
+ogd_theme_base_url = 'http://opendata.swiss/themes/'
+mapping_groups_dict = {
+    'arbeit-und-erwerb': ['work'],
+    'finanzen': ['finances'],
+    'preise': ['prices'],
+    'tourismus': ['tourism'],
+    'volkswirtschaft': ['national-economy'],
+    'wirtschaft': ['national-economy'],
+    'verwaltung': ['administration'],
+    'soziales': ['social-security'],
+    'freizeit': ['territory'],
+    'umwelt': ['territory'],
+    'politik': ['politics'],
+    'mobilitat': ['mobility'],
+    'kultur': ['culture'],
+    'kriminalitat': ['crime', 'public-order'],
+    'gesundheit': ['health'],
+    'basiskarten': ['territory', 'geography'],
+    'energie': ['energy'],
+    'bildung': ['education'],
+    'bevolkerung': ['population'],
+    'bauen-und-wohnen': ['construction'],
+}
+mapping_rights_dict = {
+    'cc-by-sa': 'NonCommercialAllow - CommercialAllow - ReferenceRequired',
+    'cc-zero': 'NonCommercialAllow - CommercialAllow - ReferenceNotRequired',
+}
+
+# Values 'sporadisch oder unregelmaessig' and 'keines'
+# do not have an equivalent in OGD and are ignored
+mapping_accrualPerdiodicty = {
+    'halbjaehrlich': 'semiannual',
+    'jaehrlich': 'annual',
+    'laufend': 'continuous',
+    'monatlich': 'monthly',
+    'quartalsweise': 'quarterly',
+    'stuendlich': 'continuous',
+    'taeglich': 'daily',
+    'woechentlich': 'weekly',
+}
+
+ckan_locale_default = pylons.config.get('ckan.locale_default', None)
+
+class StadtzhSwissDcatProfile(RDFProfile):
+
+    def _rights(self, ckan_license_id):
+        return mapping_rights_dict.get(ckan_license_id)
+
+    def _themes(self, group_id):
+        return mapping_groups_dict.get(group_id)
+
+    def graph_from_dataset(self, dataset_dict, dataset_ref):
+
+        g = self.g
+
+        catalog_node = BNode()
+        dataset_node = BNode()
+        g.add((catalog_node, RDF.type, DCAT.Catalog))
+        g.add((catalog_node, DCAT.dataset, dataset_node))
+        g.add((dataset_node, RDF.type, DCAT.Dataset))
+
+        for prefix, namespace in namespaces.iteritems():
+            g.bind(prefix, namespace)
+
+        # Basic fields
+        basic_items = [
+            ('url', DCAT.landingPage, None),
+            ('version', OWL.versionInfo, ['dcat_version']),
+        ]
+        self._add_triples_from_dict(dataset_dict, dataset_node, basic_items)
+
+        # Language
+        g.add((dataset_node, DCT.language, Literal(ckan_locale_default)))
+
+        # Basic date fields
+        date_items = [
+            ('metadata_modified', DCT.modified, None),
+            ('metadata_created', DCT.issued, None),
+        ]
+        self._add_date_triples_from_dict(dataset_dict, dataset_node, date_items)
+
+        # Organization
+        organization_id = pylons.config.get('ckan.organization_id', None)
```

Comment:
Please use a more descriptive name for this config option and maybe use ckanext.stadtzh-theme as a prefix (we already have `ckanext.stadtzh-theme.descr_file`, I know not the best name either :wink: ). I'd suggest `ckanext.stadtzh-theme.dcat_ap_organization_slug`. WDYT?